### ISSUE TITLE
Members' Directory - button for members to opt in/out disappeared, also option to tag interests

### ISSUE THREAD

**Issue by mrm-git** [MantaRayMedia/bgs#455]

**Comment by mrm-git** [comment]
➤ Matty commented:

Hi @amybrew 

I'll chat with @russurwin about this one.

**Comment by mrm-git** [comment]
➤ Russ Urwin commented:

@amybrew @mattys it looks like someone has made some change to the Civi profile https://www.bgs.org.uk/civicrm/admin/uf/group/update?action=update&id=24&context=group  can you check with Geraint/Veda to see if this is just a mistake?  The 'View/Edit Drupal User Account ' checkbox is set in our staging environment but not on live

**Comment by mrm-git** [comment]
➤ Amy commented:

@geraintcollingridge1 Could you check this at the Civi end?

**Comment by mrm-git** [comment]
➤ Geraint Collingridge commented:

No changes by BGS - will ask Veda
Just to confirm you want it to be changed to expose publically? As in screenshot attached

**Comment by mrm-git** [comment]
➤ Amy commented:

@russurwin @mattys @geraintcollingridge1 - where are we with this currently? Are we waiting on anything and can I help?

**Comment by mrm-git** [comment]
➤ Geraint Collingridge commented:

@amybrew with Veda and Disha at this time

**Comment by mrm-git** [comment]
➤ Amy commented:

@geraintcollingridge1 OK thanks for clarifying!

**Comment by Mattys** [comment]
Hey, you guys keep tagging me instead of @Matty but I'm not part of your project ^^

**Comment by matty** [comment]
Nothing to do with me!

**Comment by mrm-git** [comment]
➤ Geraint Collingridge commented:

@matty @amybrew 
From veda We will not be able to integrate with CiviCRM (contacts, cases, etc) without webform (as its own content type - not just enabled on the ‘Form’ Content type.

Can you also check with them if by disabling the webform content type, did they remove the following two webforms? 

https://bgs.org.uk/abstracts

https://bgs.org.uk/judges/abstract

**Comment by mrm-git** [comment]
➤ Russ Urwin commented:

@matty @geraintcollingridge1 @amybrew I've created the content type webform this was never there in our drupal installation as we normally use 'form'.  I'm not sure if abstracts or judges/abstract ever existed in our version of the site but I may be wrong on this..

**Comment by mrm-git** [comment]
➤ Geraint Collingridge commented:

@russurwin The work we’d already done on this appears to have been deleted by MRM. Is there anyway they can get this back? 

Work we had already completed is documented here: 

https://support.vedaconsulting.co.uk/issues/7393
https://support.vedaconsulting.co.uk/issues/9098
